### ISSUE TITLE
Some function names violate JavaScript conventions

### ISSUE THREAD

**Issue by magwo** [chilts/awssum#105]
Hello!

Looks like a nice library here. :)

I noticed from the examples that some non-constructor functions have a leading capital letter, which is generally discouraged for JavaScript code. JavaScript expert Douglas Crockford recommends using leading capital only on functions that require the usage of the "new" keyword. One example is: s3.CreateBucket. This is to avoid the sometimes confusing bugs that arise when you forget the new keyword in a call.

Is there a reason/background for the usage of leading capital letter function names?

**Comment by chilts** [comment]
Hi Magnus,

Yes, you're probably right but I decided that the operation names should follow what the operation names are. Once I did that with a few services it ended up being all 26 of them.

I should have probably also made the first letter of each parameter lowercase too, but again it's uppercase to reflect what they are called in AWS.

I probably won't change it now since it's 26 services x 30 operations/service x uncountable parameters which are all being used by a lot of people - so it adds up to a huge amount of backwards incompatible changes.

Unfortunate but true. :(

Thanks for pointing it out. On the flip side, I think that people are now used to how the operations are named and they know that it's not a constructor (at least in _this_ library).

Cheers,
Andy
